includes/class-taxjar-customer-record.php

```diff
@@ -184,36 +184,8 @@ public function get_data_from_object() {
 		$customer_data = array();
 
 		$customer_data[ 'customer_id' ] = $this->get_customer_id();
+		$customer_data[ 'name' ] = $this->get_customer_name();
```

Comment:
This portion was refactored into its own method to improve the readability of this method.